src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -34,6 +34,9 @@ def load_arguments(self, _):
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
         c.argument('disable_auto_upgrade', options_list=['--disable-auto-upgrade'], action='store_true', help='Flag to disable auto upgrade of arc agents.')
         c.argument('cl_oid', options_list=['--custom-locations-oid'], help="OID of 'custom-locations' app")
+        c.argument('container_registry_repository', options_list=['--pcr-repository'], help='Custom container registry repository.')
+        c.argument('container_registry_username', options_list=['--pcr-username'], help='Custom container registry repository username.')
+        c.argument('container_registry_password', options_list=['--pcr-password'], help='Custom container registry repository password.')
```

Comment:
PCR is not widely recognized terminology and we can't use that for the parameter name. let's use --container-registry-repository, --container-registry-username, and --container-registry-password as specified in the PM spec. If that's too long char count wise and not allowed by CLI, let's leave out container and use --registry-repository, --registry-username and --registry-repository